docs/api-reference/core/first-person-view.md

```diff
@@ -19,13 +19,13 @@ new FirstPersonView({id, ...});
 
 To render, a `FirstPersonView` needs to be combined with a `viewState` object with the following parameters:
 
-- `longitude` (`Number`, optional) - longitude of the camera
-- `latitude` (`Number`, optional) - latitude of the camera
-* `position` (`Number[3]`, optional) - meter offsets of the camera from the lng-lat anchor point. Default `[0, 0, 0]`.
-* `bearing` (`Number`, optional) - bearing angle in degrees. Default `0` (north).
-* `pitch` (`Number`, optional) - pitch angle in degrees. Default `0` (horizontal).
-- `maxPitch` (`Number`, optional) - max pitch angle. Default `90` (down).
-- `minPitch` (`Number`, optional) - min pitch angle. Default `-90` (up).
+- `longitude` (Number, optional) - longitude of the camera
```

Comment:
Nit: I like the backtick for actual types, but totally up to you. 

Just a thought: As part of the "typescript-ification" work, the rest of the frameworks are aligning on typescript conventions docs, which is lower case (`number`, `boolean`, `string` etc). Assuming deck wants to go the same way, we could choose to start updating them here as well - if we are changing them for other reasons.

Reply:
We do not use the tick format anywhere else. It messes with the generation of hash links for headers.

I think upgrading to typescript notions is a good idea. It will take much more effort though to parse through all existing API docs.